test/Microsoft.Framework.DependencyInjection.Tests/Utils/MultiServiceHelpers.cs

```diff
@@ -25,6 +25,11 @@ public static IEnumerable GetMultiService(Type collectionType, Func<Type, IEnume
 
         private static IEnumerable Cast([NotNull] IEnumerable collection, Type castItemsTo)
         {
+            if (collection == null)
```

Comment:
@henkmollema I don't we need the condition in this case, while ```collection``` parameter was already decorated with ```[NotNull]``` attribute
/cc @Eilon